### PR TITLE
feat(skym5-server): fixed errors

### PR DESCRIPTION
**Pull request by Kl1ck9r**

### CODE REVIEW

File: skymp5-server/cpp/server_guest_lib/WorldState.cpp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#include "WorldState.h"
+�   #include "WorldState.h"
```
Comment (d0gge) [inline skymp5-server/cpp/server_guest_lib/WorldState.cpp:1]: Typo in include directive. It's the reason of failed build.

File: skymp5-server/cpp/papyrus_vm_lib/ActivePexInstance.cpp [hunk 2]
```diff
@@ -6,6 +6,8 @@
 #include <functional>
 #include <sstream>
 #include <stdexcept>
+#include <stdexcept>
+
```
Comment (nic11) [inline skymp5-server/cpp/papyrus_vm_lib/ActivePexInstance.cpp:10]: It was already included

File: skymp5-server/cpp/papyrus_vm_lib/ActivePexInstance.cpp [hunk 3]
```diff
@@ -142,8 +145,7 @@ VarValue CastToString(const VarValue& var)
       }
     }
     case VarValue::kType_Identifier:
-      assert(false);
-      return VarValue();
+      throw std::runtime_error("Error");
```
Comment (nic11) [inline skymp5-server/cpp/papyrus_vm_lib/ActivePexInstance.cpp:148]: Please write something meaningful everywhere, not just "Error"

File: skymp5-server/cpp/papyrus_vm_lib/ActivePexInstance.cpp [hunk 4]
```diff
@@ -167,7 +169,7 @@ VarValue CastToString(const VarValue& var)
     case VarValue::kType_BoolArray:
       return GetElementsArrayAtString(var, var.kType_BoolArray);
     default:
-      assert(false);
+      throw std::runtime_error("Error");
       return VarValue();
```
Comment (nic11) [inline skymp5-server/cpp/papyrus_vm_lib/ActivePexInstance.cpp:173]: return can also be removed here

File: skymp5-server/cpp/papyrus_vm_lib/ActivePexInstance.cpp [hunk 5]
```diff
@@ -202,7 +204,7 @@ VarValue GetElementsArrayAtString(const VarValue& array, uint8_t type)
         break;
       }
       default:
-        assert(false);
+        throw std::runtime_error("Wrong Type ");
```
Comment (nic11) [inline skymp5-server/cpp/papyrus_vm_lib/ActivePexInstance.cpp:207]: Something like 'GetElementsArrayAtString: wrong type inside of array'

File: skymp5-server/cpp/papyrus_vm_lib/ActivePexInstance.cpp [hunk 6]
```diff
@@ -265,7 +267,7 @@ bool ActivePexInstance::EnsureCallResultIsSynchronous(
 
   ctx->needReturn = true;
   ctx->returnValue = VarValue(currentFnPr);
-  return false;
+  throw std::runtime_error("Error");
```
Comment (nic11) [inline skymp5-server/cpp/papyrus_vm_lib/ActivePexInstance.cpp:270]: Not an error

File: skyrim-platform/src/platform_lib/Validators.cpp [hunk 7]
```diff
@@ -1,4 +1,6 @@
 #include "Validators.h"
+#include <stdexcept>
+
```
Comment (nic11) [inline skyrim-platform/src/platform_lib/Validators.cpp:3]: There are no other changes in this file

File: skymp5-server/cpp/server_guest_lib/WorldState.cpp [hunk 8]
```diff
@@ -310,7 +311,7 @@ bool WorldState::AttachEspmRecord(const espm::CombineBrowser& br,
   } else {
     if (!locationalData) {
       logger->error("Anomally: refr without locationalData");
-      return false;
+      throw std::runtime_error("Error in 312 string ");
```
Comment (nic11) [inline skymp5-server/cpp/server_guest_lib/WorldState.cpp:314]: 1. What is 312 string?
2. Every `return false` is a normal flow here

File: skymp5-server/cpp/papyrus_vm_lib/ActivePexInstance.cpp [hunk 9]
```diff
@@ -20,7 +19,7 @@ ActivePexInstance::ActivePexInstance()
   this->parentVM = nullptr;
 }
 
-ActivePexInstance::ActivePexInstance(
+ActivePexInstance::ActivePexInstance()
```
Comment (d0gge) [inline skymp5-server/cpp/papyrus_vm_lib/ActivePexInstance.cpp:22]: This should be:
```
ActivePexInstance::ActivePexInstance()
{
  this->parentVM = nullptr;
}
```

**Pull request closed without merging**